### DISCUSSION TITLE
QTable column sort does not triggerring data model to sort

### DISCUSSION THREAD

**Discussion by 549393092** [quasarframework/quasar#15891] [General - Components / Directives / etc] [answered]
Dear Quasar team,

I would like to sort data model when user sort table column.
However, it seems the data model still keep the original order. Could you please check below demo?
https://codepen.io/Stanley-549393092/pen/LYgqGeb?editors=1011
How can I achive that? 

Thanks!

**Comment by suleimanas** [comment, marked as answer]
Hello,
sure you can. by using filteredSortedRows computed prop

like:
```
<q-table ref="table" ... >
..
</q-table>
```

and in your test method
```
  methods: {
    test () {
      console.log(this.$refs.table.filteredSortedRows)
    }
  }
```

**Reply by 549393092** [reply]
@suleimanas Thanks a lot! It works.
